### ISSUE TITLE
react-lite is not compatible with last version of collapse

### ISSUE THREAD

**Issue by blackswanny** [nkbt/react-collapse#167]
after updating to the last version of react-collapse, the next code started to fail referring to rect-collapse component in the end. React-lite says it's something in babel and ES6 not to be changed.
react-lite '0.15.34'
react-collapse '3.3.0'
```
function initVcomponent(vcomponent, parentContext, namespaceURI) {
    var Component = vcomponent.type;
    var props = vcomponent.props;
    var uid = vcomponent.uid;

    var componentContext = getContextByTypes(parentContext, Component.contextTypes);
    var component = new Component(props, componentContext);
    var updater = component.$updater;
    var cache = component.$cache;

    cache.parentContext = parentContext;

react-lite.common.js:433 Uncaught TypeError: Cannot set property 'parentContext' of undefined
```

**Comment by nkbt** [comment]
I have no idea about react-lite and cannot help you unfortunately.

May it be related to the switch to PureComponent and ES6 classes? I may rollback those changes and publish major version instead to avoid breaking on minor release...

**Comment by nkbt** [comment]
Patch version 3.3.1 just published. So if you rely on API of older version of react - update to it and it should work as usual.

Major version 4.0.1 published with breaking change (dropping React <15.3 support)

Please reopen if any issues are still there!
